core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -41,16 +56,15 @@ public static UnderFileSystemConfiguration defaults() {
   private UnderFileSystemConfiguration() {
     mReadOnly = false;
     mShared = false;
-    mUfsConf = Collections.EMPTY_MAP;
+    mUfsConf = new InstancedConfiguration(Configuration.copyProperties());
   }
 
   /**
    * @param key property key
    * @return true if the key is contained in the given UFS configuration or global configuration
    */
   public boolean containsKey(PropertyKey key) {
```

Comment:
shall we change this to `isSet` too? Fine to leave this in a separate PR to minimize the footprint